OrderTimeOutService/OrderTimeOut.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using DataAccess.Data;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Host;
+using Microsoft.Extensions.Logging;
+
+namespace OrderTimeOutService
+{
+    public  class OrderTimeOut
+    {
+        private readonly BarDataBase _barDataBase;
```

Comment:
This works.

But if we are using repository approach, i think an Order repository should be referenced here. It is very important to be super consistent in your application to avoid unexpected regressions later on